python-frontend/src/main/java/org/sonar/python/SubscriptionVisitor.java

```diff
@@ -140,6 +155,16 @@ public PythonFile pythonFile() {
       return pythonVisitorContext.pythonFile();
     }
 
+    @Override
+    public Map<String, Set<Symbol>> getTypeShedSymbols() {
+      return SubscriptionVisitor.getTypeShedSymbols();
```

Comment:
Why can't we return `Typeshed.typeShedSymbols` and remove the static methods on SubscriptionVisitor?